### PR TITLE
Adding the "lipo" package

### PR DESCRIPTION
**Pull request by jdb78**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jdb78** [comment]
This PR is ready for review @conda-forge-admin, please ping team

File: recipes/lipo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<36]
+
+requirements:
+  host:
+    - pip
+    - python
+    - scikit-learn >=0.22.1
```
Comment (synapticarbors) [inline recipes/lipo/meta.yaml:21]: With `poetry`, do you need anything beyond `poetry`, `pip` and `python`?

File: recipes/lipo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a943adcda8be0326774ac9a1fcca20b955f9746e58ef00586d145b018fd90ee4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<36]
```
Comment (synapticarbors) [inline recipes/lipo/meta.yaml:15]: I think you can get rid of the `skip` and make this `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

and then just change `python` to `python >=3.6` under `host` and `run` below

File: recipes/lipo/meta.yaml (hunk 1 above)
Comment (jdb78) [reply recipes/lipo/meta.yaml]: true. fixed that

File: recipes/lipo/meta.yaml (hunk 2 above)
Comment (jdb78) [reply recipes/lipo/meta.yaml]: Right. Thanks for the pointer!

**Comment by jdb78** [comment]
Many thanks for the review and your commitment to conda-forge!! I addressed your comments and this PR is again ready for review @conda-forge-admin, please ping team

File: recipes/lipo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - poetry >=1.0.5
+  run:
+    - python
```
Comment (synapticarbors) [inline recipes/lipo/meta.yaml:23]: Can you also change this to `python >=3.6`

Comment (jdb78) [reply recipes/lipo/meta.yaml]: Overseen this. Done.

**Comment by jdb78** [comment]
Thanks! I had overseen the missing python dependency specification. The PR is again ready for review @conda-forge-admin, please ping team

**Comment by synapticarbors** [comment]
@jdb78 -- thank you for your contribution to conda-forge. Once the feedstock for this package is created, please don't hesitate to contact us if you run into issues.

**Pull request merged**